upstream-community-operators/kubefed-operator/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -281,71 +285,129 @@ spec:
     quickly get up and running with this powerful concept. Federation is a key
     part of any Hybrid Cloud capability.
 
-    **Important Note**: This operator is purely for prototyping purposes and not
-    supported by any upstream community at this time.
-
-
-    **Important Note**: Currently, while in preview, this operator does not
-    support automatic upgrades. You must remove the old version of the operator
+    **Support**: 
+    
+    The KubeFed operator is purely for prototyping purposes and not
+    supported by any upstream community at this time. Currently, 
+    while in preview, this operator does not support automatic 
+    upgrades. You must remove the old version of the operator
     manually before installing a new version.
 
 
-    ## Using Federation
-
-    This deploys Federation in a namespace-scoped configuration which handles
-    the resources only in the namespace where federation is deployed. For
-    example, if you subscribe to this package in namespace `my-namespace`,
-    federation will be deployed to manage objects _only_ in `my-namespace` in
-    the host cluster and target clusters.
-
-    After installing Federation instance, you will use the `kubefedctl` command
-    line tool to do two types of configuration:
-      
-    - Configure federation to connect to your target clusters with `kubefedctl join`
-    - Create new federation APIs for the API types you want to federate with `kubefedctl enable`
-
-
-    **Important Note**: To add federation capability to your namespace, you are
-    only required to install kubefed in the hosting cluster; there is no need
-    to install kubefed in the target clusters (and no effect gained from
-    doing so).
+    ## Start using the operator
 
+    The KubeFed operator supports deploying a KubeFed control plane in both
+    namespace-scoped and cluster-scoped modes. For example, if you subscribe to
+    this package in a namespace `my-namespace`, KubeFed will be deployed to
+    manage objects _only_ in `my-namespace` in the host cluster and target
+    clusters.
+    
+    The operator requires the `KubeFed` CRD to deploy a KubeFed control plane in a given cluster.
```

Comment:
Should it also talk about the CR itself here?

Reply:
Do you mean a one-liner description?

Reply:
I meant to say we should also talk about the operator reconciling the CR instance once the user creates it in order to deploy the control plane.